cli/tauri.js/src/api/dependency-manager/util.ts

```diff
@@ -5,9 +5,19 @@ import { sync as crossSpawnSync } from 'cross-spawn'
 import { appDir, resolve as appResolve } from '../../helpers/app-paths'
 import { existsSync } from 'fs'
 import semver from 'semver'
+import { lookpath } from 'lookpath'
 
 const BASE_URL = 'https://docs.rs/crate/'
 
+async function useYarn(): Promise<boolean> {
+  const hasYarnLockfile = existsSync(appResolve.app('yarn.lock'))
+  if (hasYarnLockfile) {
+    return true
+  } else {
+    return (await lookpath('npm')) === undefined
```

Comment:
What happens if the user has neither yarn nor npm installed? Shouldn't an error be thrown? It seems like it will just assume yarn if npm is not installed.